content/influxdb/cloud-dedicated/query-data/execute-queries/optimize-queries.md

```diff
@@ -29,7 +31,25 @@ Use the following tools to help you identify performance bottlenecks and trouble
   - [Inspect Flight response headers](#inspect-flight-response-headers)
 - [Retrieve query information](#retrieve-query-information)
 
-<!-- /TOC -->
+
+## Strategies for improving query performance
+
+A query may be slow due to the following reasons:
+
+- It queries a large time-range of data.
+- It includes intensive operations, such as `ORDER BY` on large amounts of data.
+- The query plan isn't optimal--for example, applying the same sort (`ORDER BY`) to already sorted data.
+- It needs to retrieve many parquet files from object storage. The same query performs better if it retrieves fewer - though, larger - files.
+- It queries many overlapped parquet files.
+- It queries many string values. A query against a field that stores integers outperforms a query against string data.
+
+Follow these strategies to help improve query performance:
+
+- Follow [schema design best practices](/influxdb/cloud-dedicated/write-data/best-practices/schema-design/) to make your data easier to query.
+- [Downsample data](/influxdb/cloud-dedicated/process-data/downsample/).
+- Use custom-partitioning for your data. (How?)
+
+- If resource usage and costs allow, “prewarm” query caches by running the query a few times.
```

Comment:
- Only works if your memory is allocated enough for  a large cache. For Serverless, cache may have changed.